spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require "json-matchers"
+require "json_matchers"
 require "json/matchers"
```

Comment:
This ensures that all flavors of the require are tested.

If they're working properly, the first should evaluate, while the rest return `false` and no-op.

Is there a better way?